### PR TITLE
contracts/*: golint updates for this or self warning

### PR DESCRIPTION
**Pull request by kielbarry**
In the contracts directory, I have updated the variable names for receivers generating a golint warning regarding use of generic names like this or self.

### CODE REVIEW

File: contracts/ens/ens.go [hunk 1]
```diff
@@ -100,45 +100,45 @@ func ensNode(name string) common.Hash {
 	return crypto.Keccak256Hash(parentNode[:], parentLabel[:])
 }
 
-func (self *ENS) getResolver(node [32]byte) (*contract.PublicResolverSession, error) {
-	resolverAddr, err := self.Resolver(node)
+func (s *ENS) getResolver(node [32]byte) (*contract.PublicResolverSession, error) {
```
Comment (karalabe) [inline contracts/ens/ens.go]: Please swap `s` to `ens`. ENS is already an abbreviation, let's not abbreviate it further :)

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 64ed58a3a7ccd6a37ddd25c0d857a095a7772238
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

File: contracts/ens/ens.go [hunk 2]
```diff
@@ -153,26 +153,26 @@ func (self *ENS) Resolve(name string) (common.Hash, error) {
...
-func (self *ENS) Register(name string) (*types.Transaction, error) {
+func (ens *ENS) Register(name string) (*types.Transaction, error) {
 	parentNode, label := ensParentNode(name)
-	registrar, err := self.getRegistrar(parentNode)
+	registrar, err := ens.getRegistrar(parentNode)
 	if err != nil {
 		return nil, err
 	}
-	return registrar.Contract.Register(&self.TransactOpts, label, self.TransactOpts.From)
+	return registrar.Contract.Register(&ens.TransactOpts, label, ens.TransactOptens.From)
```
Comment (karalabe) [inline contracts/ens/ens.go]: `TransactOptens`?

File: contracts/ens/ens.go [hunk 3]
```diff
@@ -153,26 +153,26 @@ func (self *ENS) Resolve(name string) (common.Hash, error) {
...
-	resolver, err := self.getResolver(node)
+	resolver, err := ens.getResolver(node)
 	if err != nil {
 		return nil, err
 	}
 
-	opts := self.TransactOpts
-	opts.GasLimit = 200000
+	opts := ens.TransactOpts
+	optens.GasLimit = 200000
```
Comment (karalabe) [inline contracts/ens/ens.go]: `optens`?

**Comment by intizarabe** [comment]
As I got the situation we have to finalise block , on betting base

**Comment by intizarabe** [comment]
I need to discuss the technical matters, the other side it us clear to issue the signal for crowdsale and voting via Eth Ecosystem. Pls any comment if is right

File: contracts/ens/ens.go (hunk 2 above)
Comment (kielbarry) [reply contracts/ens/ens.go]: A mistake I am correcting now.

File: contracts/ens/ens.go (hunk 3 above)
Comment (kielbarry) [reply contracts/ens/ens.go]: A mistake I am correcting now.

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 64ed58a3a7ccd6a37ddd25c0d857a095a7772238
  * Commits must be prefixed with the package(s) they modify

 * Commit: 1bcc642259ecce0e4660d404f29bdc48f057308a
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 64ed58a3a7ccd6a37ddd25c0d857a095a7772238
  * Commits must be prefixed with the package(s) they modify

 * Commit: 1bcc642259ecce0e4660d404f29bdc48f057308a
  * Commits must be prefixed with the package(s) they modify

 * Commit: 25f893be4449495ae0e8cb61ea58cf122e9a7c5b
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 64ed58a3a7ccd6a37ddd25c0d857a095a7772238
  * Commits must be prefixed with the package(s) they modify

 * Commit: 1bcc642259ecce0e4660d404f29bdc48f057308a
  * Commits must be prefixed with the package(s) they modify

 * Commit: 25f893be4449495ae0e8cb61ea58cf122e9a7c5b
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Comment by kielbarry** [comment]
@karalabe apologies for this being stale. I've made the changes you requested and am passing the CI/CD build - please let me know if there are additional steps I need to take.

**Pull request merged**